ui-v2/app/components/sort-control.js

```diff
@@ -0,0 +1,19 @@
+import Component from '@ember/component';
+import { get, set } from '@ember/object';
+export default Component.extend({
+  classNames: ['sort-control'],
+  direction: 'asc',
+  onchange: function() {},
+  init: function() {
+    this._super(...arguments);
+    this.guid = this.elementId;
```

Comment:
what is this `guid` used for?

Reply:
ah! good catch, a variable from a past iteration, will remove thanks!